app/services/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -112,8 +112,17 @@ def current_date(format: '%F')
           apipie :method, 'Checks whether a value is truthy or not' do
             optional :value, Object, desc: 'Value to check'
             returns one_of: [true, false], desc: 'Returns true if the value can be considered as truthy, false otherwise'
+            example "truthy?(true) #=> true"
+            example "truthy?(false) #=> false"
+            example "truthy?('true') #=> true"
+            example "truthy?('false') #=> false"
+            example "truthy?(1) #=> true"
+            example "truthy?(0) #=> false"
             example "truthy?('1') #=> true"
             example "truthy?('0') #=> false"
+            example "truthy?('some-string') #=> false"
```

Comment:
Although from Ruby perspective it should be `true` :/

Reply:
I agree, I noticed that too, we had a discussion about it with @ares, I was suggesting to use `ActiveRecord::Type::Boolean.new.deserialize` method instead of `Foreman::Cast.to_bool` because of this case, but that could case discrepancy with the logic from other methods like `host_param_true?`.

Reply:
Could we pls open a PR dropping `Foreman::Cast.to_bool` (deprecating it) and use the `ActiveRecord::Type::Boolean.new.deserialize` instead? We should not implement Rails tooling (we will not do a better job at it ^_^).

It is out of scope of this PR and definitelly not a blocker as this is ready to get in, just saying it would be nice to open up that discussion while we are hitting it.

Reply:
Created an RM for this not to block a PR on it: [RM#34424](https://projects.theforeman.org/issues/34424)